substrate/frame/revive/src/lib.rs

```diff
@@ -1327,15 +1327,20 @@ where
 		Self::convert_evm_to_native(fee, ConversionPrecision::RoundUp)
 	}
 
+	/// Convert a weight to a gas value.
+	pub fn evm_weight_to_gas(weight: Weight) -> U256 {
```

Comment:
Nit: The name is a bit confusing, because we are converting the substrate weight to EVM gas.

Reply:
yeah right, wanted to prefix all these functions with evm_lbut you are right it indeed reads "evm_weight", will come up with something better